### PR TITLE
Cleanup Travis-CI configuration and install VIPS for image processing

### PR DESCRIPTION
**Pull request by Manfred**
* Use package syntax to install packages
* Just use yarn to install JavaScript dependencies
* Install VIPS from source because Active Storage requires a recent build

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,16 +5,38 @@ rvm:
...
 cache:
-- bundler
-- yarn
-- npm
+  bundler: true
+  yarn: true
+  directories:
+  - $HOME/vips
 before_install:
-- npm i -g node@latest
```
Comment (sudara) [inline .travis.yml]: This used to be required, but seems like everything is happy now?

**Comment by sudara** [comment]
Do we know what triggered master going red?

Are there any cons to libvips I should know about? I haven't done my homework there...

**Comment by sudara** [comment]
I approved Percy, one diff was intermittent (date related) to be fixed and the other was intentional change from Ben on master.

File: .travis.yml (hunk 1 above)
Comment (Manfred) [reply .travis.yml]: Yeah, I think Ubuntu used to ship with an ancient version of Node. But probably good enough for now.

**Comment by Manfred** [comment]
Thijs said the reason for the failure was trying to use NPM And Yarn simultaneously. Not really sure why that would cause libsass to stop building.

Cons for VIPS is that we need a recent version and there is no package on Ubuntu. Speed and less reliance on crufty ImageMagick is the primary reason for switching. The ImageProcessing gem used by Rails 6 can use either so we can always switch back.

**Pull request merged**